src/Goerli-DssSpell.t.sol

```diff
@@ -323,14 +171,14 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         assertTrue(lerp.done());
     }
 
-    function testNewChainlogValues() public { // make public to use
+    function testNewChainlogValues() private { // make public to use
```

Comment:
just a note on specific test for new chainlog entries (currently disabled), as mentioned in chat, if it should be discarded in favour of the new chainlog tests or kept as sanity check on key-value and version of the new entries.
I'd keep it as a sanity check to ensure key-value and version are set as expected.

Reply:
The new test does the same thing so long as the chainlog entry makes it into the addresses file.  I think I'm okay with removing the old test.

Reply:
No, this was privatized in the scrub from last week. I should still add a test for the new values in particular. The new tests will only check that our local copy has all of the keys that are on chainlog. If we add or update it will warn that we haven't changed the version, but it doesn't check for inclusion.

Reply:
These were added.